packages/chat-component/src/components/chat-thread-component.ts

```diff
@@ -41,6 +41,12 @@ export class ChatThreadComponent extends LitElement {
   @property({ type: Object })
   selectedCitation: Citation | undefined = undefined;
 
+  @property({ type: Boolean })
```

Comment:
since you are not using the branding in the thread this can be removed. Same with the icon below.

Reply:
Done!